proposals/0390-noncopyable-structs-and-enums.md

```diff
@@ -1575,6 +1590,35 @@ We can expose the ability for code to implement these coroutines directly,
 which is a good optimization for copyable value types, but also allows for
 more expressivity with noncopyable properties.
 
+### Casting function types based on ownership modifiers
+The current rules for casting functions with a noncopyable parameter is that
+the ownership modifier must remain the same. 
+
+One reason behind this limitation is a matter of scope. There is a broader need
+to support such casts even for copyable types. For example, a function
+containing an `inout` parameter cannot currently be cast to or from any other
+function type, unless if the parameter remains `inout`. But it should be safe to
```

Comment:
```suggestion
function type, unless the parameter remains `inout`. It should be safe to
```

Reply:
The "cannot.... unless..." construction would be more readable by describing only what is currently supported, than describing what _could_ be supported. My suggestion doesn't achieve that.

Reply:
Thanks. I'm just gonna drop the sentence / point since it's not adding much anyway.